hackclub.com.yaml

```diff
@@ -2465,10 +2465,15 @@ hupy:
     type: CNAME
     value: hupy.github.io.
 
-
+codethetech:
+- ttl: 1
+  type: CNAME
+  value: codethetech.github.io
+    
```

Comment:
There doesn't seem to be a github pages here, and I think this site is out-of-scope for hackclub.com, @grymmy can you chime in on scope?